source/wp-content/themes/wporg-documentation-2022/parts/header-navigation.html

```diff
@@ -0,0 +1,8 @@
+<!-- wp:navigation {"textColor":"white","backgroundColor":"charcoal-2","className":"is-style-dropdown-on-mobile","style":{"spacing":{"blockGap":"24px"}},"fontSize":"small"} -->
```

Comment:
I think it might be better to generate this nav with PHP, so that it can be translated & so the dynamic `.current-menu-item` class can be added. Maybe the approach used on [wporg-main-2022](https://github.com/WordPress/wporg-main-2022/blob/trunk/source/wp-content/themes/wporg-main-2022/src/navigation/index.php) could be reused, or moved up to wporg-mu-plugins?

Reply:
Tried adding to mu-plugins and using here and it seems to work nicely. I'll raise a PR 👍